andela/wger-mugiwara
# #144207593 exercise cache when deleting muscles.

#### What does this PR do?
Ensures cache is updated when deleting muscles.
#### Description of task to be completed.
The cache should be reset after a muscle has been deleted. Muscle should not appear after it has been deleted.
#### How should this be manually tested?
After cloning the repo, run the local host server using the command `python manage.py runserver` on your terminal.

- On the web page, register and login with your user account.

- Click on the Exercises drop down button, which will give you a list of options. Click on muscles.

- It will redirect you to a new page, on the top right, click on add muscle the add any name of a muscle of your preference.

- After adding the muscle, go back to the Exercises drop down options and click on Muscle overview. The muscle you added should appear on the page you are redirected to.

- Go back and delete the muscle from the Muscles page. The muscle should not appear in Muscles and Muscle overview page.
